File: pocs/android_only_m2_j17jars/app/src/main/java/bisq/android/impl/AndroidMemoryReportService.java
package bisq.android.impl;

import java.util.concurrent.CompletableFuture;

import bisq.common.platform.MemoryReportService;

public class AndroidMemoryReportService implements MemoryReportService {
    @Override
    public void logReport() {
    }

    @Override
    public long getUsedMemoryInBytes() {
        return 0;
    }

    @Override
    public long getUsedMemoryInMB() {
        return 0;
    }

    @Override
    public long getFreeMemoryInMB() {
        return 0;
    }

    @Override
    public long getTotalMemoryInMB() {
        return 0;
    }

    @Override
    public CompletableFuture<Boolean> initialize() {
        return null;
    }

    @Override
    public CompletableFuture<Boolean> shutdown() {
        return null;
    }
}

File: pocs/android_only_m2_j17jars/app/src/main/java/bisq/android/impl/AndroidMemoryReportService.kt
package bisq.android.impl

import bisq.common.platform.MemoryReportService
import java.util.concurrent.CompletableFuture

class AndroidMemoryReportService : MemoryReportService {
    override fun logReport() {
    }

    override fun getUsedMemoryInBytes(): Long {
        return 0
    }

    override fun getUsedMemoryInMB(): Long {
        return 0
    }

    override fun getFreeMemoryInMB(): Long {
        return 0
    }

    override fun getTotalMemoryInMB(): Long {
        return 0
    }

    override fun initialize(): CompletableFuture<Boolean>? {
        return null
    }

    override fun shutdown(): CompletableFuture<Boolean>? {
        return null
    }
}
